### DISCUSSION TITLE
Cannot save end entity profile after upgrade to 7.11.0

### DISCUSSION THREAD

**Discussion by tlull** [Keyfactor/ejbca-ce#233] [Q&A] [answered]
Hello,
after upgrading from 7.4.3.2 to 7.11.0 I'm not able to save end entity profiles for personal certificates.
I get the following error message:

`"A SAN rfc822name field with domain or list of domains as default value must be modifiable."`

But I haven't defined any "Subject Alternative Name" attribute.

Kind regards,
   Torsten

**Comment by primetomas** [comment]
Do you have an email field? Or a subject DN email field?

**Comment by tlull** [comment]
Yes, I have an "End Entity E-mail" field and I have this field "emailAddress, E-mail address in DN" within the DN.

**Comment by primetomas** [comment]
So the message can also relate to the DN email field. Play with that.

**Comment by tlull** [comment, marked as answer]
After removing "emailAddress, E-mail address in DN" and adding it again I can save the end entity profile. This is a weird behaviour....
Thanks for your hint, the error message is misleading.
